src/api/types.jl

```diff
@@ -263,10 +349,19 @@ const H5L_TYPE_HARD    = 0
 const H5L_TYPE_SOFT    = 1
 const H5L_TYPE_EXTERNAL= 2
 
+# H5O_INFO constants
+const H5O_INFO_BASIC = Cuint(0x0001)
+const H5O_INFO_TIME = Cuint(0x0002)
+const H5O_INFO_NUM_ATTRS = Cuint(0x0004)
+const H5O_INFO_HDR = Cuint(0x0008)
+const H5O_INFO_META_SIZE = Cuint(0x0010)
+const H5O_INFO_ALL = H5O_INFO_BASIC | H5O_INFO_TIME | H5O_INFO_NUM_ATTRS | H5O_INFO_HDR | H5O_INFO_META_SIZE
```

Comment:
Shouldn't this just be 
```
H5O_INFO_BASIC | H5O_INFO_TIME | H5O_INFO_NUM_ATTRS
```

 as per 
https://github.com/HDFGroup/hdf5/blob/develop/src/H5Opublic.h#L89

seems like  you also folded in 
https://github.com/HDFGroup/hdf5/blob/develop/src/H5Opublic.h#L98

Reply:
It depends on which version of the library and function we are using:

`H5O_GET_INFO2` and `H5O_GET_INFO3` says `H5O_INFO_ALL` should be `H5O_INFO_BASIC | H5O_INFO_TIME | H5O_INFO_NUM_ATTRS | H5O_INFO_HDR | H5O_INFO_META_SIZE`

https://portal.hdfgroup.org/display/HDF5/H5O_GET_INFO2
https://portal.hdfgroup.org/display/HDF5/H5O_GET_INFO3

https://github.com/HDFGroup/hdf5/blob/1d90890a7b38834074169ce56720b7ea7f4b01ae/src/H5Opublic.h#L91